README.rst

```diff
@@ -9,6 +9,10 @@ any solution of their choosing.
 HaaS keeps track of available resources in a database, which a system
 administrator must populate initially. 
 
+This project is part of the larger Massachusetts Open Cloud, for a list
+of authors, description of the team, development workflow, etc. see
+``https://github.com/CCI-MOC/moc-public/blob/master/README.md``.
+
```

Comment:
Put this somewhere where it's not in the middle of a separate topic -- the paragraphs above and below this are related to eachother, don't split them up.